simulation/halsim_gui/src/main/native/cpp/DriverStationGui.cpp

```diff
@@ -426,7 +426,7 @@ void GlfwSystemJoystick::GetData(HALJoystickData* data, bool mapGamepad) const {
 
   // copy into HAL structures
   data->desc.isGamepad = m_isGamepad ? 1 : 0;
-  data->desc.type = m_isGamepad ? 21 : 20;
+  data->desc.gamepadType = m_isGamepad ? 21 : 20;
```

Comment:
Are these mappings still accurate? `HAL_GetJoystickGamepadType` and `DriverStation:: GetJoystickGamepadType` are documented to map to [SDL_GamepadType](https://wiki.libsdl.org/SDL3/SDL_GamepadType), but that doesn't seem to have any values for 21 or 20.